app/cdap/services/metadata-parser/index.js

```diff
@@ -115,6 +115,7 @@ function createApplicationObj(entityObj) {
     version,
```

Comment:
So I don't want to touch this `version` because i think it's meant to be artifact version

Reply:
put a comment then